frontend/src/main/scala/bloop/dap/JavaDebugInterface.scala

```diff
@@ -0,0 +1,60 @@
+package bloop.dap
+
+import java.net.{URL, URLClassLoader}
+
+import bloop.exec.JavaEnv
+import bloop.io.{AbsolutePath, RelativePath}
+
+import scala.util.{Failure, Success, Try}
+
+object JavaDebugInterface {
+
+  /**
+   * Checks if the Java Debug Interface is either already on classpath or if it can be automatically loaded.
+   */
+  def isAvailable: Try[_] = {
+    def loadJdi: Try[_] = Try(Class.forName("com.sun.jdi.Value"))
+
+    def addToClasspath(jar: AbsolutePath): Unit = {
```

Comment:
As discussed privately, I think all of this should be unnecessary because JDI is by default always accessible if the host JVM is a JDK. If that's not the case, I'd assume there's a problem with the installed JDK. I'm a little bit afraid of doing this `addURL` magic and I'm willing to bet this is not compatible with Java 9 and Java 11 where JDI is a module instead. The best is probably to remove all of this unnecesary logic and provide excellent error messages to users so that they don't feel lost if their Java machine is not a JDK.

Reply:
OK, "works on my machine" so I will drop this functionality